.github/workflows/wheels.yml

```diff
@@ -19,23 +20,18 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04, windows-latest, macos-latest]
+        os: [ubuntu-20.04, windows-latest, macos-latest]
```

Comment:
By the way, you probably should pin all or use latest on all, rather than a mix. This likely was from an example in cibw that was like this but is now consistent.